Binaries.csv

```diff
@@ -52,6 +52,6 @@ Textures,UT3NightShadeTex.utx,https://cloud.greatemerald.eu/s/RnyiwYng5rXLJbT/do
 Textures,UT3PaladinTex.utx,https://cloud.greatemerald.eu/s/rR0SDFe2qOANAEJ/download,da945b45505e93dbb73819ac3f423b5be3c73084f2074d1d0095fd69086c681d,
 Textures,UT3RaptorTex.utx,https://cloud.greatemerald.eu/s/1EYdvMC3cB087XP/download,caa46e1c57dd5b7b9fe5055094cc97dbdec6b15bd1c8f419f4348e91412d64d1,
 Textures,UT3ScorpionTex.utx,https://cloud.greatemerald.eu/s/Ib2uqx9hrjY89ZL/download,37bde4d6a3fc84481d94853e58ebb2c71341381926d44843ac088bc106f4d818,
-Textures,UT3SPMATex.utx,https://cloud.greatemerald.eu/s/B97CV0O36eNE6KI/download,4d1bc0b533b7e067ad524357dca8a9ff2c3b1a1b5945115215463237a88ceffb,
+Textures,UT3SPMATex.utx,https://drive.google.com/uc?export=download&id=1TCHrscY6o8gpEI9ptxV7alp7v11X5RSq,f89e8cde48eab301b6834fa3795b593bc75fa3c80065664a5697585c787716e6,
```

Comment:
This needs `--content-disposition`.

Reply:
Ah right forgot about that, Alex, place it at the end of the line, it's needed for G-Drive files only